app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -11,6 +11,7 @@ CONFIG_COMP_CHAIN_DMA=y
 CONFIG_COMP_CROSSOVER=y
 CONFIG_COMP_DRC=y
 CONFIG_COMP_KPB=y
+CONFIG_COMP_TESTER=m
```

Comment:
Aa, right, so this is in separate commit, but the one change to mtl/modules.conf was left to a different commit?